patches/chromium/feat_add_support_for_missing_dialog_features_to_shell_dialogs.patch

```diff
@@ -282,15 +282,27 @@ index 9853bac93511ff5f790833528829a2c0017ed505..8fd2b4bc632bc15667e58b333a50312a
    if (!service_started.value_or(false)) {
      g_service_availability = ServiceAvailability::kNotAvailable;
      return;
-@@ -175,17 +176,21 @@ void SelectFileDialogLinuxPortal::StartAvailabilityTestInBackground() {
+@@ -120,9 +121,12 @@ void OnServiceStarted(std::optional<bool> service_started) {
+ }
+ 
+ void OnSystemdUnitStarted(dbus_xdg::SystemdUnitStatus) {
++#if 0
+   dbus_utils::CheckForServiceAndStart(dbus_thread_linux::GetSharedSessionBus(),
+                                       kXdgPortalService,
+                                       base::BindOnce(&OnServiceStarted));
++#endif
++  file_dialog::StartPortalAvailabilityTestInBackground();
+ }
+ 
+ DbusByteArray PathToByteArray(const base::FilePath& path) {
+@@ -175,17 +179,20 @@ void SelectFileDialogLinuxPortal::StartAvailabilityTestInBackground() {
    GetMainTaskRunner() = base::SequencedTaskRunner::GetCurrentDefault();
  
    dbus_xdg::SetSystemdScopeUnitNameForXdgPortal(
 -      dbus_thread_linux::GetSharedSessionBus().get(),
 -      base::BindOnce(&OnSystemdUnitStarted));
 +    dbus_thread_linux::GetSharedSessionBus().get(),
```

Comment:
It is very likely the shared session bus usage here will bring back the bug https://github.com/electron/electron/issues/46652#issuecomment-2846088555, I will test and create a follow-up to avoid blocking this PR.